config/thoth.yaml

```diff
@@ -6,9 +6,14 @@ repositories:
     managers:
       - name: update
         configuration:
-          labels: bot
+          labels: [bot]
       - name: info
       - name: version
+        configuration:
+          maintainers:
```

Comment:
could we keep stuff like that in an OWNERS file within the repository itself?

Reply:
Sure, we can. Is it still used by Sesheta?

Reply:
no, right now it is not